keyboards/preonic/keymaps/trigotometry/keymap.c

```diff
@@ -0,0 +1,204 @@
+#include "preonic.h"
+#include "action_layer.h"
```

Comment:
Could you remove all of these additional includes?  

They get included automatically, at various points in the built process, so they are redundant and not necessary here.